documentation/docs/steps/cloudFoundryCreateService.md

```diff
@@ -10,28 +21,158 @@
-            cloudFoundry: [apiEndpoint: 'https://test.server.com',
-                credentialsId: 'cfCredentialsId',
-                serviceManifest: 'manifest-create-service.yml',
-                org: 'cfOrg',
-                space: 'cfSpace'])
+    cfApiEndpoint : 'https://test.server.com',
+    cfOrg : 'cfOrg',
+    cfSpace: 'cfSpace',
+    cfCredentialsId: 'cfCredentialsId',
+    cfService:  'myService',
+    cfServiceInstanceName: 'myServiceInstanceName',
+    cfServicePlan: 'myPlan',
+    cfCreateServiceConfig: '{\"example\":\"value\",\"example\":\"value\"}',
+    cfServiceTags: 'list, of, tags',
+    script: this,
+)
+```
+
+If you chose to having a dedicated JSON file for the JSON configuration for the `cfCreateServiceConfig` flag you can do so by referencing the file path accordingly. This file should be stored in the same folder as your `Jenkinsfile` that starts the Pipeline in order for the Pipeline to be able to find the file. Most favourable SCM is Git.
+Such a JSON file with the appropriate step configuration could look as follows:
+
+The JSON config file, e.g. `createServiceConfig.json` can look like this:
+
+```json
+{
+  "example":"value",
+  "example":"value"
+}
 ```
 
-The following example additionally to above also makes use of a variable substitution file `mainfest-variable-substitution.yml`.
+The step configuration needs to contain the path to the JSON file:
 
 ```groovy
 cloudFoundryCreateService(
-            script: this,
-            cloudFoundry: [apiEndpoint: 'https://test.server.com',
-                credentialsId: 'cfCredentialsId',
-                serviceManifest: 'manifest-create-service.yml',
-                manifestVariablesFiles: ['mainfest-variable-substitution.yml'],
-                org: 'cfOrg',
-                space: 'cfSpace'])
+    cfApiEndpoint : 'https://test.server.com',
+    cfOrg : 'cfOrg',
+    cfSpace: 'cfSpace',
+    cfCredentialsId: 'cfCredentialsId',
+    cfService:  'myService',
+    cfServiceInstanceName: 'myServiceInstanceName',
+    cfServicePlan: 'myPlan',
+    cfCreateServiceConfig: 'createServiceConfig.json',
+    cfServiceTags: 'list, of, tags',
+    script: this,
+)
+```
+
+* ### Multiple Service Creation in Cloud Foundry example with manifest file in Jenkinsfile
+
+The following example shows the option to create multiple Services in Cloud Foundry. It makes use of the Cloud Foundry Create-Service-Push Plugin. This is described in above Prerequisites, please check this section for further information regarding its usage. This plugin enables this step to create multiple Cloud Foundry Services in one step.
 
+It requires a dedicated YAML file, e.g. `manifest.yml`, that contains all the information for creating the services, including their names, service plan and the service broker.
+
+Such a `manifest.yml` file needs to have the following structure:
+
+```yaml
+
+---
+create-services:
+- name:   "testService1"
+  broker: "testBroker"
+  plan:   "testPlan"
```

Comment:
It would be great to include a real example (probably a widely known one - e.g. mongoDB)

Reply:
Good hint - mongoDB has been added as an example.